src/Glpi/Console/Migration/FormCreatorPluginToCoreCommand.php

```diff
@@ -0,0 +1,179 @@
+
+class FormCreatorPluginToCoreCommand extends AbstractCommand
+{
+    /**
+     * Error code returned if plugin version or plugin data is invalid.
+     *
+     * @var integer
+     */
+    public const ERROR_PLUGIN_VERSION_OR_DATA_INVALID = 1;
+
+    /**
+     * Error code returned if import failed.
+     *
+     * @var integer
+     */
+    public const ERROR_PLUGIN_IMPORT_FAILED = 1;
+
+    /**
+     * Version of Formcreator plugin required for this migration.
+     * @var string
+     */
+    public const FORMCREATOR_REQUIRED_VERSION = '2.13.9';
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('migration:formcreator_plugin_to_core');
+        $this->setDescription(__('Migrate Formcreator plugin data into GLPI core tables'));
+
+        $this->addOption(
+            'skip-errors',
+            's',
+            InputOption::VALUE_NONE,
+            __('Do not stop on import errors')
+        );
+
+        $this->addOption(
+            'truncate',
+            't',
+            InputOption::VALUE_NONE,
+            __('Remove existing core data')
+        );
+
+        $this->addOption(
+            'without-plugin',
+            'w',
+            InputOption::VALUE_NONE,
+            sprintf(
+                __('Enable migration without plugin files (we cannot validate that plugin data are compatible with supported %s version)'),
+                self::FORMCREATOR_REQUIRED_VERSION
+            )
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $migrationManager = new MigrationManager($this->db, $this);
+        if (!$migrationManager->checkPlugin(!$input->getOption('without-plugin'))) {
+            return self::ERROR_PLUGIN_VERSION_OR_DATA_INVALID;
+        }
+
+        if ($input->getOption('truncate')) {
+            $this->cleanCoreTables();
+        }
+
+        if (!$this->migratePlugin()) {
+            return self::ERROR_PLUGIN_IMPORT_FAILED;
+        }
+
+        $output->writeln('<info>' . __('Migration done.') . '</info>');
+
+        return 0; // Success
+    }
+
+    /**
+     * Clean data from core tables.
+     *
+     * @throws RuntimeException
+     */
+    private function cleanCoreTables()
+    {
+        $core_tables = [
+            'glpi_forms_accesscontrols_formaccesscontrols',
+            'glpi_forms_answerssets',
+            'glpi_forms_categories',
+            'glpi_forms_comments',
+            'glpi_forms_destinations_answerssets_formdestinationitems',
+            'glpi_forms_destinations_formdestinations',
+            'glpi_forms_forms',
+            'glpi_forms_questions',
+            'glpi_forms_sections',
+            'glpi_plugin_formcreator_forms_groups',
+            'glpi_plugin_formcreator_forms_languages',
+            'glpi_plugin_formcreator_forms_profiles',
+            'glpi_plugin_formcreator_forms_users',
+            'glpi_plugin_formcreator_forms_validators'
```

Comment:
This does not seems like core tables, is this temporary code because there are not yet supported ?
In this case we could add a comment with a TODO.